junit-mixin-plugin-test/src/test/scala/org/scalajs/junit/TraitToClassMixinTest.scala

```diff
@@ -0,0 +1,24 @@
+package org.scalajs.junit
+
+import org.junit._
+import org.junit.Assert._
+
+object TraitToClassMixinTest {
+  var count = 0
```

Comment:
It is not set in `beforeClass` this design would not be compatible with `TraitToObjectMixinTest`.